test/ansible/roles/bmh_setup/tasks/main.yaml

```
@@ -0,0 +1,50 @@
+---
+# create libvirt BMH network with dns server
+- name: create libvirt network
+  shell: export LOADBALANCER_IP=$(cat /tmp/loadbalancer_ip) && cat "{{ src_dir}}/test/e2e/libvirt/network-bmh.xml.tpl" | sed "s/<LOADBALANCER_IP>/${LOADBALANCER_IP}/g" >/tmp/libvirt-network-bmh.xml && (virsh net-destroy bmh || true) && (virsh net-undefine bmh || true) && virsh net-define /tmp/libvirt-network-bmh.xml
```

```suggestion
  shell: |
    export LOADBALANCER_IP=$(cat /tmp/loadbalancer_ip)
    cat "{{ src_dir}}/test/e2e/libvirt/network-bmh.xml.tpl" | sed "s/<LOADBALANCER_IP>/${LOADBALANCER_IP}/g" >/tmp/libvirt-network-bmh.xml
    virsh net-destroy bmh || true
    virsh net-undefine bmh || true
    virsh net-define /tmp/libvirt-network-bmh.xml
```

could be more readable?